README.markdown

```diff
@@ -28,7 +28,7 @@ To use a custom timeout, create an initializer file:
 
 ```ruby
 # config/initializers/rack_timeout.rb
-Rack::Timeout.timeout = 5  # seconds
+Rack::Timeout.service_timeout = 5  # seconds
```

Comment:
It [is compatible](https://github.com/heroku/rack-timeout/blob/init-settings/lib/rack/timeout/legacy.rb#L12). I just want the readme to reflect the new preferred setting.